spec/koans.js

```diff
@@ -8,31 +8,31 @@ describe('`let` restricts the scope of the variable to the current block - ', ()
   describe('`let` vs. `var`.', () => {
     it('`var` works as usual, it does not restricts scope', () => {
       if (true) {
-        /*You should add your code in here*/
+       varX = true;
       }
-      // expect(varX).toBe(true);
+      expect(varX).toBe(true);
     });
 
     it('`let` restricts scope to inside the block', () => {
-      /*var or const? letX = false*/
+      letX= false/*var or const? letX = false*/
       if (true) {
-        /*var or const? letX = true*/
+       constX= true /*var or const? letX = true*/
       }
-      //expect(letX).toBe(false);
+      expect(letX).toBe(false);
     });
 
     it('`var` does not restricts scope to inside the block in `for` loops', () => {
-      /*var or let? counter = 100*/
-      /*for (var or let? counter = 1; counter < 50; counter++){}*/
+      //let counter = 100 /*var or let? counter = 100*/
+     for (var counter = 1; counter < 50; counter++){};
 
-      //expect(counter).toBe(50);
+      expect(counter).toBe(50);
     });
 
     it('`let` restricts scope to inside the block also in `for` loops', () => {
-      /*var or let? counter = 100*/
+      let counter = 100
       /*for (var or let? counter = 1; counter < 50; counter++){}*/
 
-      //expect(counter).toBe(100);
+      expect(counter).toBe(100);
     });
```

Comment:
Let is block scoped, that means that if you change its value inside an if statement or a for loop, its value will not be changed outside of this for loop (or this if statement), so you can use let both times here : 
```js
    it('`let` restricts scope to inside the block also in `for` loops', () => {
      let counter = 100;
      for (let counter = 1; counter < 50; counter++){}

      expect(counter).toBe(100);
    }); 
  });
```